website_mrp_bom/__init__.py

```diff
@@ -0,0 +1,2 @@
+from . import models
```

Comment:
Missing generic Copyright

Reply:
I'd rather not to put it in that tiny files